File: 2011. Final Value of Variable After Performing Operations.py
class Solution:
    def finalValueAfterOperations(self, operations):
        x = 0
        for i in range(0, len(operations)):
            if "-" in operations[i]:
                x -= 1
            else:
                x += 1
        return x


newFinalValue = Solution()
print(newFinalValue.finalValueAfterOperations(["--X", "X++", "X++"]))

File: 2011_final-value-of-variable-after-performing-operations.c
#include <stdio.h>

int finalValueAfterOperations(char **operations, int operationsSize)
{
    int X = 0;
    for (int i = 0; i < operationsSize; i++)
        operations[i][1] == '-' ? X-- : X++;

    return X;
}

int main()

{
    char *operations[] = {"X++", "++X", "--X", "X--"};
    int operationsSize = sizeof(operations) / sizeof(operations[0]);

    int X = finalValueAfterOperations(operations, operationsSize);
    printf("%d", X);

    return 0;
}
